spec/sms_spec.rb

```diff
@@ -0,0 +1,9 @@
+ require 'Sms'
+
+ describe Sms do
+ it "receives a text message after the order is placed" do
+    VCR.use_cassette('twilio') do
+      subject.send
+    end
+  end
```

Comment:
Lint/BlockAlignment: `end` at 8, 2 is not aligned with `it "receives a text message after the order is placed" do` at 4, 1.